docs/content/docs/pub-sub-implementations.md

```diff
@@ -146,12 +146,10 @@ marshaler := kafka.NewWithPartitioningMarshaler(func(topic string, msg *message.
 
 ### HTTP
 
-At this moment only HTTP subscriber is available. There is an issue for a [HTTP publisher](https://github.com/ThreeDotsLabs/watermill/issues/17).
-
-HTTP subscriber allows us to send messages received by HTTP request (for example - webhooks).
+The HTTP subscriber listens to HTTP requests (for example - webhooks) and outputs them as messages.
```

Comment:
`  [HTTP]({{< ref "#http" >}})  |   | x | `prod-ready` |` should be changed at the bottom of the page (add publisher)

Reply:
988ec6531abbd023c5fcba55b153e3c8ba953f13